scripts/qa_crowbarsetup.sh

```diff
@@ -1863,7 +1867,7 @@ function onadmin_allocate
     else
         if [[ $hacloud = 1 ]] ; then
             cluster_node_assignment
-            local nodes=($unclustered_nodes)
+            local nodes=(${unclustered_nodes[@]})
```

Comment:
You're right, I was sure I checked that but looks like I didn't. The syntax with double conversion seems to be the official one. Bash is full of surprises :)

On top of that, to handle items with spaces properly you might want to add quotes like this: `local nodes=("${unclustered_nodes[@]")`.